code/numerical_analysis/integral/src/integral_rectangle.java

```diff
@@ -0,0 +1,33 @@
+package cosmos;
+
+//Part of Cosmos by OpenGenus Foundation
+public class IntegralRectangle {
+
+	public static void main(String[] args) {
+		
+		double a = 0.0;
+		double b = 1.0;
+		int step = 1000;
+
+		IntegralFunction integralFunction = (x) -> x * x * x; // a∫b x^3 dx
+		double sum = calculateIntegral(a, b, step, integralFunction);
+		System.out.println("Approximate integral value = " + sum); // exact answer = 0.25
+	}
+
+	public static double calculateIntegral(double a, double b, int numOfRectangles, IntegralFunction function) {
```

Comment:
You may want to split each parameter to a specific line.

Reply:
What do you actually mean by "_split each parameter to a specific line_"?

Reply:
Eg:

```java
public static double calculateIntegral(
   double a, 
   double b, 
   int numOfRectangles, 
   IntegralFunction function) 
{
    // ...
}
```

This would shorten the line, and make it easier to understand.